specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/MaintenanceConfigurations.json

```
@@ -0,0 +1,360 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-10-02-preview",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/managedEnvironments/{environmentName}/maintenanceConfigurations": {
+      "get": {
+        "tags": [
+          "MaintenanceConfigurations"
+        ],
+        "operationId": "MaintenanceConfigurations_ListByManagedEnvironmentResource",
```

```suggestion
        "operationId": "MaintenanceConfigurations_List",
```
MaintenanceConfigurations is sub resource of managed environment and there is only one list. Prefer naming as above. Then the generated operation name will be list() in MaintenanceConfigurations Resource file, which align with other sub resources like storage and cert